### ISSUE TITLE
Connect to specific servers by IP address (slither-nest functionality)

### ISSUE THREAD

**Issue by clemens-tolboom** [ermiyaeskandary/Slither.io-bot#283]
Asked in https://github.com/ErmiyaEskandary/Slither.io-bot/issues/275#issuecomment-223628208 to 'port'  https://github.com/LukeeeeBennett/slither-nest
- [ ] Add two preference variables `window.serverConnection` and `window.serverConnectionKeep`
- [ ] Add UI checkbox when checked use the same server binds to `window.serverConnectionKeep`
- [ ] Add UI textbox for editing server binds to `window.serverConnection`

**Comment by ermiyaeskandary** [comment]
I'm doing the custom server connect functionality - :+1: 
Much easier way

**Comment by clemens-tolboom** [comment]
I had this in mind. No fuss

![slither_io](https://cloud.githubusercontent.com/assets/371014/15798211/a039340e-2a31-11e6-8f21-ba2d432166b5.png)

``` javascript
            oContent.push('<input type="checkbox" title="Connect to same server"/><input type="text" title="Game server" size="20" placeholder="111.222.333.444:555" pattern="((^|\.)((25[0-5])|(2[0-4]\d)|(1\d\d)|([1-9]?\d))){4}$"/>' );
```

When playing we can set the textbox and when check it keeps connecting to same arena.
When not playing one can edit the textbox / checkbox

Please make the IP/port a bot variable so others can reach it/set it too. Like my C&C

Pattern is incomplete and was found on http://html5pattern.com/Miscs
Somehow the z-index (or something else) is not good as I cannot edit it.

**Comment by ermiyaeskandary** [comment]
> Somehow the z-index (or something else) is not good as I cannot edit it.

You know why ?
Because of where you put it - 
`// Set static display options here.`

**Comment by clemens-tolboom** [comment]
> // Set static display options here.

lol ... that's not quite true. 'onPrefChange' is zapped on mouse clicks.

I added some code to branch feature/283-fixed-server-option (quite bad code)

### Questions
- [ ] how to integratie with default settings?
- [ ] when to set the values?
- [ ] I added jQuery as a dependency which would make lots of stuff easier. I have no problem with jQuery unless we use it in a tight loop for updating lots.

**Comment by clemens-tolboom** [comment]
Updated the summary and made my https://github.com/ErmiyaEskandary/Slither.io-bot/issues/283#issuecomment-224625333 more stand out.

**Comment by ermiyaeskandary** [comment]
@clemens-tolboom We should do it without jQuery... An extra dependency just makes it slower and it's possible. We should do it on the screen with the play button not within the overlays... like slither-nest.

**Comment by ermiyaeskandary** [comment]
@clemens-tolboom We have to set the values after we die
`force-server` is a built in slither.io function which forces a server after death.
I'll double check and report back.

**Comment by ermiyaeskandary** [comment]
@clemens-tolboom We can integrate with default settings by saving it after it is set and loading it before we play the game - but then run it after each death. I need to double check `force-server` but you should only need to set it once with that function and it would stay there until page refresh. I disagree with saving it as a setting anyway. We would want to switch servers regularly and we can always set it to something else manually after refresh - we just want the bots to be on the same server when testing. Which server it is doesn't matter TBH.

**Comment by clemens-tolboom** [comment]
@ErmiyaEskandary I'm still learning. I did some coding to get this issue started. Please take over :p 
(I know it is assigned to you but was struggling today with my C&C so needed a break).

**Comment by clemens-tolboom** [comment]
Is your solution scriptable? And without the extra button?
![slither_io](https://cloud.githubusercontent.com/assets/371014/15903820/e37e4044-2dae-11e6-899b-44eff0e8e5b4.png)

**Comment by ermiyaeskandary** [comment]
@clemens-tolboom Yeah - we can just put an input box. I can start working on this soon but really busy currently :( 
Thanks for having a go by the way!
